lightbulb/command_handler.py

```diff
@@ -169,6 +175,33 @@ def __init__(
 
         self._help_impl = help_class(self)
 
+        self.extensions_path = extensions_path
+        if extensions_path:
+            self.subscribe(hikari.StartedEvent, self.start_extension_watch)
+
+    async def start_extension_watch(self, _) -> None:
+        async for changes in awatch(self.extensions_path):
+            for change in changes:
+                change_type = change[0]
+                change_path = change[1]
+
+                file_changed = (change_path.split(os.sep)[-1]).split(".")[0]
+                extension_path = self.extensions_path.replace(os.sep, ".") + "." + file_changed
+
+                try:
+                    if change_type == Change.deleted:
+                        self.unload_extension(extension_path)
+
+                    if change_type == Change.modified:
+                        self.reload_extension(extension_path)
```

Comment:
I dont get why not loading extensions. That would make it really easy to just drag and drop extensions into the extensions folder.